libnmstate/netapplier.py

```diff
@@ -28,7 +28,6 @@
 from libnmstate.nm.nmclient import nmclient_context
 from libnmstate import state
 from libnmstate import validator
-from libnmstate.deprecation import _warn_keyword_as_positional
```

Comment:
Please use the commit message template, squash this commit and the one for updating the integration tests for this change and consider using "Adwait Thattey <coderdude1999@gmail.com>" for the signed-of-by tag.